source/partials/release_notes/_release-17-9-0.md.erb

```diff
@@ -0,0 +1,74 @@
+<h4>Improvements</h4>
+* <%= link_to_commit '928c5856c2bc73aa92c1a8eb1a86d8094a9fe806', 'Grouping elastic profile by plugin-id', 3772%>
+
+  This provides an organized view of the elastic profiles making it easier to find a particular elastic agent.
+
+  <figure class='shrink-figure-70'>
+      <%= image_tag 'releases/17.9.0/elastic-agent-grouping-by-plugin-id.png', alt: 'Group elastic profile by plugin-id' %>
+  </figure>
+
+* <%= link_to_commit 'ca7c3fa232087b67553e7d30bbfb409ba5aab49d', 'Addition of a request processor to allow plugins to query the server info', 3401%>
+
+* <%= link_to_commit '1c1f2fe0d4ebda46ddfe0766ce50d580039a3f77', 'Admins can go to pipelines edit page directly from VSM'%>
+
+  <figure class='shrink-figure-70'>
+      <%= image_tag 'releases/17.9.0/edit-pipeline-from-vsm.png', alt: 'Edit pipeline from VSM' %>
+  </figure>
+
+* <%= link_to_commit 'd47180f2193239ddd318157c43d3f60b6168d9e0', 'Addition of a few more fields to materials form which is used in creation of pipelines.'%>
+
+<h4>API Changes</h4>
+* <%= link_to_commit '0587f21042701d9ee1e64ae698341c495df6c8d8', 'Plugin info api v3', 3719%>
+* <%= link_to_commit '0f7e98110e2b3443534227c53a1bd663dc52a592', 'Config repo api v1', 3656%>
+
+<h4>Bug Fixes</h4>
+* <%= link_to_commit 'e538ba0032fcdd6cc8c95d202267deeb6c46a979', 'Check if agent is disabled before job assignment'%>
+* <%= link_to_commit '8f6cc6c8c52fabecae024600d454e9a094d9fc2a', 'Send request headers from server to plugins'%>
+* <%= link_to_commit '0eab696075a5bbdf7cd117a1392101559705bab1', 'Stage and job related warnings should not be logged for a rerun stage/job', 3753%>
+* <%= link_to_commit '8f66499e169fdb3b8ff74b40da1f816bab7716dc', 'Fixes select all bug on admin edit in environment page', 3737%>
+* <%= link_to_commit '793d19809f2693028ab40ef83403810ef47a3b1f', 'Redirect authenticated user to home page on re-authentication'%>
+
+<h4>Docker Images</h4>
+* Log server output and logs to STDOUT in addition to writing logs to log files, so you can now watch all server logs using docker logs.
```

Comment:
Need some clarity on what this means. Which change specifically does this relate to?

Reply:
@nyuday , while using docker containers, all the data within it (which includes logs) perish once the container stops. Supposing there's an error while starting up the server or agent, GoCD logs it, but these logs are lost because the container stops. This change to the docker images ensures that the logs are output to stdout which is visible to the user upon starting the container. These logs will be present when users do `docker logs <container_id>` for them to debug.

Reply:
@yankurk , its not just server output. Its also agent's.

Reply:
Thanks @varshavaradarajan  
Suggested heading: Docker Image and Log Enhancements
-We've launched the GoCD Agent docker image for the latest version of Alpine 3.6.
-Logs related to Docker containers can now be accessed even after the container stops using  docker logs <container_id>

Reply:
> -Logs related to Docker containers can now be accessed even after the container stops using docker logs <container_id>

That's just default docker behavior, not something we've implemented.

Reply:
Let me clarify that a bit, the "logs can be access after container stops" is default docker behavior. Earlier, because we were not sending logs to docker, users could not see the logs, now that we do, users can see the logs in a manner they are familiar with.

Reply:
Is this better?
GoCD now sends logs to Docker so you can access logs related to Docker containers anytime.

Reply:
> GoCD now sends logs to Docker so users can access logs related to Docker containers anytime.

Much much better. Thanks, I'd probably tweak it with some information about how to get to the logs "GoCD now sends logs to docker (by writing them to STDOUT) so users can access the logs anytime by executing `docker logs`"

Reply:
Sounds great. Minor Edit - GoCD now sends logs to docker by writing them to STDOUT. Users can access the logs anytime by executing docker logs